src/libraries/System.Text.RegularExpressions/tests/RegexReductionTests.cs

```diff
@@ -410,8 +410,6 @@ public void PatternsReduceIdentically(string pattern1, string pattern2)
             {
                 throw new Xunit.Sdk.EqualException(result2, result1);
             }
-
-            Assert.NotEqual(GetRegexCodes(new Regex(pattern1, RegexOptions.RightToLeft)), GetRegexCodes(new Regex(pattern2)));
```

Comment:
I'm not sure I follow why we are removing the assert.

Reply:
Good question.  It was only marginally related, in that I only noticed the assert when I made a change temporarily and caused it to fail.  Which caused me to look at it, and realize that it wasn't actually testing anything valuable, so I removed it.  I'd added it in the .NET 5 timeframe, I think maybe as a control condition to validate that two patterns that should always be different compare differently, but we now have a separate test for exactly that.